Chapter_2/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+AllCops:
```

Comment:
Chapter_2 should not have the following files:

    .rubocop.yml
    .ruby-gemset
    .ruby-version
    Gemfile
    Gemfile.lock
    Guardfile
    
Please use `git rm Chapter_2/_______` to remove them.